README.rst

```diff
@@ -0,0 +1,153 @@
+=======================
```

Comment:
I think we should add a Changelog or Release file to start tracking all the changes.
We should also start naming the releases or add a description on the GitHub release page. @morenol What do you think?

Reply:
Yes, I also thing that we need a Changelog

Reply:
We can add something like this to the cookiecutter, what do you thing?

https://github.com/edx/edx-cookiecutters/blame/master/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/CHANGELOG.rst

I can add something like that to this repository

Reply:
Yes, I agree. @morenol

Reply:
Please, review that in https://github.com/eduNEXT/cookiecutter-openedx-plugin/pull/9 and then I will create other PR in this repo @Squirrel18

Reply:
Ok.